src/components/ConnectivityQuery.vue

```diff
@@ -569,6 +569,24 @@ export default defineComponent({
             store.commit.setDirectionalEdges(true);
             store.commit.setQueriedNetworkState(true);
             store.commit.setDegreeEntries(setNodeDegreeDict(store.state.networkPreFilter, store.state.networkOnLoad, store.state.queriedNetwork, store.state.directionalEdges));
+            if (promise.length >= 100) {
+              // Create dictionary of degree occurences
+              const orderedList = Object.values(store.state.nodeDegreeDict).sort((a, b) => a - b);
+              const degreeCount: {[key:number]:number} = {};
+              // eslint-disable-next-line no-plusplus
+              for (let i = 0; i < orderedList.length; i++) {
+                degreeCount[orderedList[i]] = (degreeCount[orderedList[i]] || 0) + 1;
+              }
+
+              // Set min value if the node degree occurence < 100
+              Object.entries(degreeCount).every(([degree, occurence]) => {
+                if (occurence < 100) {
+                  store.commit.setMinDegree(Number(degree));
+                  store.commit.setDegreeNetwork([Number(degree), store.state.maxDegree]);
+                  return false;
+                } return true;
+              });
```

Comment:
This should help simplify a bit. We're not using the boolean results from the `every`, so I think `forEach` is the way to go.

```suggestion
              Object.entries(degreeCount).forEach(([degree, occurrence]) => {
                if (occurrence < 100) {
                  store.commit.setMinDegree(Number(degree));
                  store.commit.setDegreeNetwork([Number(degree), store.state.maxDegree]);
                }
              });
```

Reply:
Will your suggestion keep going through all the entries? I'm using `every` so that it stops the `for loop` once the conditional is met

Reply:
Oh, that makes sense. I think your approach is better. Would you just leave a comment above the `every` call, to remind me when I see this again? :)

Reply:
yup!